tests/mailmap/basic.c

```diff
@@ -0,0 +1,88 @@
+#include "clar.h"
+#include "clar_libgit2.h"
+
+#include "common.h"
+#include "git2/mailmap.h"
+
+static git_mailmap *mailmap = NULL;
+
+const char TEST_MAILMAP[] =
+	"Foo bar <foo@bar.com> <foo@baz.com>  \n"
+	"Blatantly invalid line\n"
+	"Foo bar <foo@bar.com> <foo@bal.com>\n"
+	"<email@foo.com> <otheremail@foo.com>\n"
+	"<email@foo.com> Other Name <yetanotheremail@foo.com>\n";
```

Comment:
I don't think this one is valid. At least git-check-mailmap does not mention this syntax

Reply:
It's not mentioned, but I did test with git & this format is handled by git's internal parser.

Reply:
git's documentation doesn't mention this format, but when I was testing mailmap, IIRC this format is supported.

Reply:
Okay, fair enough